central/cve/fetcher/orchestrator.go

```diff
@@ -207,6 +212,17 @@ func (m *orchestratorCVEManager) reconcileCVEs(clusters []*storage.Cluster, cveT
 			version = metadata.GetVersion()
 		case utils.OpenShift:
 			version = metadata.GetOpenshiftVersion()
+		case utils.Istio:
+			versions, err := m.cveMatcher.GetValidIstioVersions(allAccessCtx, cluster)
+			if err != nil || len(versions) < 1 {
+				continue
+			}
+			for _, v := range versions.AsSlice() {
```

Comment:
I think you can just do `for version := range versions { ... }`

`versions` is a `Set[string]`, and a `type Set[KeyType comparable]` is just `map[KeyType]struct{}`.

You can loop through maps like the following:

```
for key, value := range myMap { ... }
for key := range myMap { ... }
for _, value := range myMap { ... }
```